packages/node_modules/samples/browser-plugin-meetings/index.html

```diff
@@ -297,7 +297,7 @@ <h2>Then click:</h2>
         <button id="destroyStats" title="stop filter getStats" type="button">stop getStats</button>
         <p>
           <label for="sendDtmfInput">Send DTMF Tones</label>
-          <input id="sendDtmfInput" name="sendDtmf" value="" placeholder="DTMF Numbers" type="number" />
+          <input id="sendDtmfInput" name="sendDtmf" value="" placeholder="DTMF Numbers" type="text" />
```

Comment:
Do we want to do some kind of validation before sending it up?

Grabbed from https://gist.github.com/d-levin/21df4ab1ad759bb7ecd865100a7909fd
`\d` = 0-9

```suggestion
          <input id="sendDtmfInput" name="sendDtmf" pattern="^[\d#*]+$" value="" placeholder="DTMF Numbers" type="text" />
```

Reply:
Per our meeting, we don't, but, I'm going to add it so that it has a tighter scope in the PR.